### ISSUE TITLE
Task: Linear Regression Model

### ISSUE THREAD

**Issue by rishisahoo** [rishisahoo/energy#24]
Train a linear regression model on the training data and predict values on the test set.
We used a linear regression model for building and training the cooling and heating load models.
Code is added to this file

Supports Story #4

**Comment by CadeKolanowski** [comment]
Subtasks: This task is used to create the Linear Regression model and predicts values using the Heating Load and Cooling Load datasets
1: model_heating = LinearRegression()
model_heating.fit(X_train_h, y_train_h)
y_pred_h = model_heating.predict(X_test_h) creates the Linear Regression model for predicting the Heating Load set by using training variables
2: model_cooling = LinearRegression()
model_cooling.fit(X_train_c, y_train_c)
y_pred_c = model_cooling.predict(X_test_c) the same is done for the Cooling Load set
